L/Libepoxy/build_tarballs.jl

```diff
@@ -39,4 +38,4 @@ dependencies = [
 ]
 
 # Build the tarballs, and possibly a `build.jl` as well.
-build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; julia_compat="1.6")
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies; clang_use_lld=false, julia_compat="1.6")
```

Comment:
Is `clang_use_lld=false` necessary?

Reply:
I needed it for macos when I tested it under https://github.com/JuliaPackaging/BinaryBuilderBase.jl/pull/426 

Perhaps it is not needed with your PR.

Reply:
Now only apple aarch64 fails.

Reply:
Meson is so stupid

Reply:
In case it wasn't clear, since meson does something dumb when lld is used as linker, the option `clang_use_lld=true` is necessary again.